src/cli.js

```diff
@@ -1,103 +1,205 @@
+        .then((c) => {
+            if (verbose) {
+                console.info(`Created ${chalk.green(c.outputFilePath)}`);
+            }
+            c.messageList.forEach((message) => {
+                const warningTitle = chalk.yellow(`WARNING: ${pathName}`);
+                const warningInfo = message;
+                handleWarning(`${warningTitle}\n${warningInfo}`);
             });
+            return c;
         })
-        .catch(reason => console.error(chalk.red('[Error] ' + reason)));
-    });
-};
+        .catch((reason) => {
+            const errorTitle = chalk.red(`ERROR: ${pathName}`);
+            const errorInfo = reason;
+            handleError(`${errorTitle}\n${errorInfo}`);
+        })
+);
 
-function getSource(file, relativeTo) {
-    return new Promise((resolve, reject) => {
-      sass.render({file}, (err, result) => {
-        if (err && relativeTo && relativeTo !== '/') {
-          return resolve([]);
+const createTypingsForFileOnWatch = (creator, cache, verbose) => (pathName) => {
+    let warnings = 0;
+    let errors = 0;
+
+    const handleError = (error) => {
+        console.error(error);
+        errors += 1;
+    };
+    const handleWarning = (warning) => {
+        console.warn(warning);
+        warnings += 1;
+    };
+    const onComplete = () => {
+        if (warnings + errors > 0) {
+            console.info(`${pathName}: ${warnings} warnings, ${errors} errors`);
         }
-  
-        if (err && (!relativeTo || relativeTo === '/')) {
-          return reject(err);
+        warnings = 0;
+        errors = 0;
+    };
+
+    return createTypings(pathName, creator, cache, handleError, handleWarning, verbose)
+        .then(onComplete);
+};
+
+const createTypingsForFiles = (creator, cache, verbose) => (pathNames) => {
+    let warnings = 0;
+    let errors = 0;
+
+    const handleError = (error) => {
+        console.error(error);
+        errors += 1;
+    };
+    const handleWarning = (warning) => {
+        console.warn(warning);
+        warnings += 1;
+    };
+    const onComplete = () => {
+        if (warnings + errors > 0) {
+            console.info(`Completed with ${warnings} warnings and ${errors} errors.`);
         }
-  
-        resolve(result.css.toString());
-      });
-    });
-  }
+        errors = 0;
+        warnings = 0;
+    };
+
+    return Promise.all(pathNames.map(
+        pathName => createTypings(pathName, creator, cache, handleError, handleWarning, verbose),
+    )).then(onComplete);
+};
 
-let main = () => {
-    let rootDir, searchDir;
+
+const main = () => {
+    const yarg = yargs.usage('Create .scss.d.ts from CSS modules *.scss files.\nUsage: $0 [options] <input directory>')
+        .example('$0 src/styles')
+        .example('$0 src -o dist')
+        .example('$0 -p styles/**/*.scss -w')
+
+        .detectLocale(false)
+        .version(pkg.version)
+
+        .demandCommand(1, 1, 'Input directory must be specified', 'Only one input directory must be specified')
```

Comment:
This forces input directory to be mandatory, so 'atsm -p styles/**/*.scss -w' fails with error. Instead we should default to ./ when input dir is not provided.